lib-injection/build/docker/java/config-1.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
```

Comment:
I guess it won't be easy, and we could do it in the future, but I think all these files are very similar, later on I could use some template system or something like that.

Reply:
+1 we should be able to template these via helm later.